src/pseudo/NormConservingPsp.jl

```diff
@@ -7,19 +7,21 @@ abstract type NormConservingPsp end
 
 #    lmax::Int                   # Maximal angular momentum in the non-local part
 #    h::Vector{Matrix{Float64}}  # Projector coupling coefficients per AM channel: h[l][i1,i2]
-#    Zion::Int                   # Ionic charge (Z - valence electrons)
```

Comment:
why?

Reply:
Better to have this as a function rather than a field. We already have a function of that name. Might was well use it also for psp structs.